actions/add-review-url/spec/addReviewUrlTest.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
```

Comment:
Be consistent in your usage of single or double quotes.

Reply:
Updated.